packages/form-builder/src/index.tsx

```diff
@@ -5,10 +5,13 @@ import {BlockSupports, registerBlockType} from '@wordpress/blocks';
 import App from './App';
 
 import sectionBlocks, {sectionBlockNames} from './blocks/section';
-import fieldBlocks from './blocks/fields';
-import elementBlocks from './blocks/elements';
+import blockRegistrar from '@givewp/form-builder/common/registrars/blocks';
 import {FieldBlock} from '@givewp/form-builder/types';
 
+import './blocks/fields';
+import './blocks/elements';
+import './blocks/extensions';
```

Comment:
Imports should return something. Otherwise this results in side effects, which can have unintended consequences.

Reply:
@kjohnson I've refactored the blocks registrar in a way that handles all the registration processes, including the `registerBlockType` step. It looks much clean now.

See: https://github.com/impress-org/givewp-next-gen/pull/179/files/59e2f3d8cfe0df6f944bbe272fd069d30189dbae..adf55856f6fec2a27d7fd123705e116e3fd7a7b5